### ISSUE TITLE
TypeError: Cannot read property 'black' of undefined

### ISSUE THREAD

**Issue by WhoAteDaCake** [apizzimenti/cli-weather#32]
Trying to run 
`weather -c -s -z NG92PA `

throws the following error 
```/usr/lib/node_modules/cli-weather/lib/tools/Files.js:23
 e.message = "The file couldn't be written to - " + chalk.bgwhite.black("sudo") + " may be needed to" +

TypeError: Cannot read property 'black' of undefined
    at /usr/lib/node_modules/cli-weather/lib/tools/Files.js:23:77
    at fs.js:1305:7
    at FSReqWrap.oncomplete (fs.js:123:15)
```
Most likely because of the -s flag, as just running `weather -c -z NG92PA ` works as expected

**Comment by pizzimathy** [comment]
Should be `chalk.bgWhite.black()`, not `chalk.bgwhite.black()`. Simple oversight, will be fixed in version 2.0.4 to be released later today. You may have to run `sudo` before running `$ weather -s`, as it's likely that your `node_modules` global folder doesn't have write permissions for your user.

**Comment by pizzimathy** [comment]
This bug was resolved on branch `bug/2.0.4/chalk` and was included [in the 2.0.4 release.](https://github.com/apizzimenti/cli-weather/releases/tag/2.0.4)
